docs/upgrade_to_tails_3x.rst

```diff
@@ -202,7 +202,35 @@ Once complete, you should see a success message:
 .. |Confirm Upgrade| image:: images/upgrade_to_tails_3x/confirm_upgrade.png
 .. |Installation Complete| image:: images/upgrade_to_tails_3x/installation_complete.png
 
-4. Verify the Upgrades
+4. Upgrade the Tails Persistence Configuration
+----------------------------------------------
+
+Due to changes in Tails 3.x it is essential you upgrade the custom persistence
+configuration previously set up for your *Admin Workstation* and *Journalist
+Workstation* drives:
+
+  #. Boot into each Tails volume.
+  #. Open a terminal |Terminal| and navigate to the root of the SecureDrop git
+     directory: ``cd ~/Persistent/securedrop``.
+  #. Fetch the latest SecureDrop sources ``git fetch --all``.
```

Comment:
As a user I would prefer to have something like
<pre>
#. cd ~/Persistent
</pre>
here and not the **Additionally, you skip the cd securedrop/ ...** below. I find it easier to follow the instructions to the letter rather than remembering exceptions to the instructions.